OWML.ModHelper/ModConfig.cs

```diff
@@ -18,6 +19,31 @@ public class ModConfig : IModConfig
         [JsonProperty("settings")]
         public Dictionary<string, object> Settings { get; set; } = new Dictionary<string, object>();
 
+        private T ConvertToEnum<T>(object value)
+        {
+            if (value is float || value is double)
+            {
+                var floatValue = Convert.ToDouble(value);
+                value = (long)Math.Round(floatValue);
```

Comment:
why not return here, but return if int?

Reply:
ah, i see. would prefer return here, at the cost of a few more (duplicated) words

Reply:
also, should 4.6 become 5? i guess this question doesn't even make sense 😁

Reply:
we're in *undefined* territory. i guess just casting (same as floor) is more intuitive tho?

Reply:
actually, yeah, I can just return here

Reply:
done that way because of slider